### ISSUE TITLE
Improve handling and skipping when term not present in Wikidata

### ISSUE THREAD

**Issue by lubianat** [lubianat/pyorcidator#49]
Sometimes a term is not on Wikidata and we'd rather just exclude it instead of going through the effort of creating a new entry. 

Not sure about how to implement: 

- special dict value ("SKIP") that is skipped?
- flag that skips any missing when running import?

**Comment by jvfe** [comment]
I prefer the second option. Having keys in our dict that don't refer to Q-Ids would be a big change from the current data model. 
Another idea is having a third option when receiving the prompt, e.g:

```
Is the QID for 'Systems Biology'  
 Q815297 - systems biology (computational and mathematical modeling of complex biological systems) ? (y/n/skip)
```

**Comment by lubianat** [comment]
Yeah, I like this option. And one could set a general "skip" flag to just
treat all of them as skips.
Tiago

**Comment by cthoyt** [comment]
Agreed, having a skip option would be nice.

**Comment by jvfe** [comment]
I believe the implementation would be more appropriate inside of add_key in [wdcuration.py](https://github.com/lubianat/wdcuration/blob/master/wdcuration/wdcuration.py), since that function is the one that generates the prompts to the use.
